defopt.py

```diff
@@ -25,6 +25,13 @@
 except ImportError:  # pragma: no cover
     from funcsigs import signature as _inspect_signature
 
+try:
+    import colorama
+except ImportError:  # pragma: no cover
+    pass
+else:
+    colorama.init()  # pragma: no cover
```

Comment:
I know this is the way colorama tells you to use it, but since defopt promises to not modify the functions given, I think it's also reasonable to expect it to have no side effects (especially when you're not even using the CLI). Instead, I think we should use `colorama.colorama_text` to wrap just the call to `parser.parse_args` inside `run`.